services/kafka.go

```diff
@@ -1,21 +1,35 @@
 package services
 
 import (
+	"encoding/json"
+	"log"
+	"os"
+	"strconv"
+	"time"
+
 	"github.com/Shopify/sarama"
+	"github.com/digicatapult/wasp-ingest-rtmp/util"
+)
+
+const (
+	KafkaTopicEnv     = "KAFKA_TOPIC"
+	KafkaPartitionEnv = "KAFKA_PARTITION"
```

Comment:
What is this for?  If we are using this to SET the amount of partitions then that is done by the `wasp-routing-service`.  In WASP our consumers always create the topic and its partitions/replicas.